cpp/proto/arcticc/pb2/in_memory_storage.proto

```diff
@@ -10,7 +10,6 @@ syntax = "proto3";
 package arcticc.pb2.in_memory_storage_pb2;
 
 message Config {
-    uint64 cache_size = 1; // if 0 => 10GB
```

Comment:
Scary to change a protobuf in a backwards incompatible way!

In this case probably fine, I doubt anyone has serialized this type of config anywhere. Doesn't seem worth the risk though, suggest just adding a comment to explain that this field is deprecated and unused instead?